core/templates/pages/topic-viewer-page/topic-header/topic-header.component.css

```
@@ -0,0 +1,66 @@
+  display: flex;
+  gap: 4px;
+}
+
+.topic-header-breadcrumbs-mobile {
+  display: none;
+}
+
+.topic-header-breadcrumbs a,
+.topic-header-breadcrumbs span {
+  color: #00645c;
+  font-family: 'Roboto', Arial, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)